flujo/infra/sandbox/docker_sandbox.py

```
@@ -0,0 +1,164 @@
+            timed_out = False
+            try:
+                await asyncio.wait_for(
+                    asyncio.to_thread(container.wait),
+                    timeout=request.timeout_s or self._timeout_s,
+                )
+            except asyncio.TimeoutError:
+                timed_out = True
+                try:
+                    container.kill()
+                except Exception:
+                    pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)